queue_services/entity-filer/src/entity_filer/filing_processors/correction.py

```diff
@@ -29,6 +31,8 @@ def process(correction_filing: Filing, filing: Dict, filing_meta: FilingMeta, bu
     original_filing = Filing.find_by_id(filing['correction']['correctedFilingId'])
     original_filing.parent_filing = correction_filing
 
+    filing_meta.correction = {}
```

Comment:
Since initiated here, can remove from downstream https://github.com/bcgov/lear/blob/main/queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/correction.py#L42